### PR TITLE
style: IDE0008 (var usage)

### PR DESCRIPTION
**Pull request by bcollamore**

### CODE REVIEW

**Review by ynsehoornenborg — CHANGES_REQUESTED** [review]
Let's enable `csharp_style_var_for_built_in_types` also, as we're actually removing many `var` statement with this PR

**Review by ynsehoornenborg — CHANGES_REQUESTED** [review]
Lat's use `csharp_style_var_for_built_in_types` also, as with this configuration we're actually removing many `var` keywords.

File: Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/DocumentThrownExceptionsCodeFixProvider.cs [hunk 1]
```diff
@@ -51,16 +51,16 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
...
 			DocumentationHelper docHelper = new(node);
-			var parts = exceptionTypeName.Split(',');
-			foreach (var part in parts)
+			string[] parts = exceptionTypeName.Split(',');
+			foreach (string part in parts)
 			{
 				docHelper.AddException(part);
 			}
-			var newComment = docHelper.CreateDocumentation();
+			Microsoft.CodeAnalysis.CSharp.Syntax.DocumentationCommentTriviaSyntax newComment = docHelper.CreateDocumentation();
```
[the remark this answers is not in the document]
Comment (bcollamore) [reply Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/DocumentThrownExceptionsCodeFixProvider.cs]: done

File: Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/DocumentThrownExceptionsCodeFixProvider.cs [hunk 2]
```diff
@@ -27,18 +27,18 @@ public sealed override FixAllProvider GetFixAllProvider()
 
 		public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 		{
-			var root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+			SyntaxNode root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
 			Diagnostic diagnostic = context.Diagnostics.First();
-			var diagnosticSpan = diagnostic.Location.SourceSpan;
+			Microsoft.CodeAnalysis.Text.TextSpan diagnosticSpan = diagnostic.Location.SourceSpan;
```
[the remark this answers is not in the document]
Comment (bcollamore) [reply Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/DocumentThrownExceptionsCodeFixProvider.cs]: done

**Comment by bcollamore** [comment]
> Lat's use `csharp_style_var_for_built_in_types` also, as with this configuration we're actually removing many `var` keywords.

done

**Review by ynsehoornenborg — APPROVED** [review]

**Pull request merged**